packages/web-client/public/images/icons/trash.svg

```diff
@@ -1,8 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="15.9" height="17.5" viewBox="0 0 15.9 17.5">
-  <g transform="translate(0.75 0.75)">
-    <path d="M3,6H17.4" transform="translate(-3 -2.8)" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/>
-    <path d="M15.844,5.2V16.4A1.575,1.575,0,0,1,14.3,18H6.549A1.575,1.575,0,0,1,5,16.4V5.2m2.324,0V3.6A1.575,1.575,0,0,1,8.873,2h3.1a1.575,1.575,0,0,1,1.549,1.6V5.2" transform="translate(-3.222 -2)" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/>
-    <path d="M10,11v5.333" transform="translate(-4.4 -4.143)" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/>
-    <path d="M14,11v5.333" transform="translate(-5.2 -4.143)" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/>
-  </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="#fff" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"><polyline points="3 6 5 6 21 6" /><path d="M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2" /><line x1="10" y1="11" x2="10" y2="17" /><line x1="14" y1="11" x2="14" y2="17" /></svg>
```

Comment:
@Aierie I took the liberty to change the trash icon to fit this feature, but this probably breaks the current upload component design. Given that that component is a prototype, are you ok if we adjust this icon when we return to the upload feature?

Reply:
Yeah, no worries on that front!